src/styl/co-styles.styl

```diff
@@ -16,6 +16,24 @@ $fontload-js ?= false
 @require 'node_modules/colette/assets/styl/_@blocks/*'
 @require 'node_modules/colette/assets/styl/_mixins/*'
 
+$lh ?= _rem(27px)
+$lh-gap ?= 2px
+$lh-division ?= .5
+
+_calcLines($lines, $font-size, $line-height = $lh)
```

Comment:
Maybe add a comment to explain the maths ?

Reply:
It is temporary waitin next release of colette.
It will be docummented in Colette